src/zulip/api.rs

```diff
@@ -96,6 +126,21 @@ impl ZulipApi {
         Ok(response)
     }
 
+    /// Is a Zulip stream public?
+    pub(crate) fn is_stream_public(&self, stream_id: u64) -> anyhow::Result<bool> {
```

Comment:
to remove two `!` around the codebase and make the code simpler, we could change this function to `is_stream_private`.

Reply:
Fixed!